src/Illuminate/Console/ConfirmableTrait.php

```diff
@@ -9,22 +9,21 @@ trait ConfirmableTrait
      *
      * This method only asks for confirmation in production.
      *
-     * @param  string  $warning
+     * @param  string|null  $warning
      * @param  \Closure|bool|null  $callback
      * @return bool
      */
-    public function confirmToProceed($warning = 'Application In Production!', $callback = null)
+    public function confirmToProceed($warning = null, $callback = null)
     {
-        $callback = is_null($callback) ? $this->getDefaultConfirmCallback() : $callback;
-
-        $shouldConfirm = value($callback);
+        $handler = $this->laravel->make(ConfirmHandler::class);
```

Comment:
this needs to resolve `ConfirmHandlerContract::class` and not the impl

Reply:
Done